src/Collectors/PublicClassMethodCollector.php

```diff
@@ -49,12 +49,6 @@ public function processNode(Node $node, Scope $scope): ?array
             return null;
         }
 
-        // skip test methods
-        $classMethodName = $node->name->toString();
-        if (str_starts_with($classMethodName, 'test')) {
-            return null;
-        }
-
```

Comment:
this is already covered by the logic which skips everything which subclasses `PHPUnit\Framework\TestCase`

Reply:
:+1: 

Could you add test for this one too?

Reply:
reverted and new tests added